### DISCUSSION TITLE
[Caching error] How to use cookiebot in next.js?

### DISCUSSION THREAD

**Discussion by Filippinifra** [vercel/next.js#29427] [Help]
Hello guys, I followed the cookiebot installation guide and now I have a problem, maybe for caching.

I put the cookiebot script in the head of the `_document` file in page's folder

<img width="866" alt="Screenshot 2021-09-27 at 13 58 33" src="https://user-images.githubusercontent.com/31794601/134904395-7f832160-6d9d-41cd-8930-be430ea72b61.png">

But when cookiebot automatically tries to fetch the "configuration file" this is what happens: 

<img width="964" alt="Screenshot 2021-09-27 at 13 59 57" src="https://user-images.githubusercontent.com/31794601/134904462-f980562c-cce2-4e60-af35-0558784a1e60.png">

So I tried to ask for this url `https://consentcdn.cookiebot.com/consentconfig/43440fdd-5912-4ff9-b2cd-8398fa372bd5/shop-smb-template-commercejs.vercel.app/configuration.js` and everything works well.

<img width="1247" alt="Screenshot 2021-09-27 at 13 59 49" src="https://user-images.githubusercontent.com/31794601/134904926-46818f21-82ec-401f-bdc9-763c658a5cde.png">

So how can I solve this? 
Why next js is caching the 404 response and is not receiving the correct one?
How can I refresh the cache?

Maybe the problem is that this call is done by the server? Should be the client to do this call?

Thanks, Francesco.

**Comment by MikeyS2002** [comment]
Did you find anything Francesco, having the same issue?

**Comment by jgray-persefoni-ai** [comment]
Any idea on a solution for this? Having similar issues utilizing Cookiebot in my NextJS app.
